migrations/2019-01-26-090348_create_version_owner_actions/up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE version_owner_actions (
+    id SERIAL PRIMARY KEY,
+    version_id INTEGER REFERENCES versions(id) ON DELETE CASCADE NOT NULL,
+    owner_id INTEGER REFERENCES users(id) NOT NULL,
```

Comment:
Do you think it might be confusing to have a column called `owner_id` pointing to the `users` table and not the `crate_owners` table? What do you think about replacing `owner` with `user` here?

Reply:
I agree with you but this is no longer in this PR. It's in a PR that's already been merged.

Would you like me to rename the field?

Reply:
I could go either way on renaming `owner_id` to `user_id`... up to you, @markcatley, if you'd like to add a migration to this PR to rename the field.